build/Dockerfile.ppc64le

```diff
@@ -16,11 +16,11 @@
 
 FROM alpine as builder
 
-RUN wget -O /qemu-ppc64le-static https://github.com/multiarch/qemu-user-static/releases/latest/download/qemu-ppc64le-static
+RUN wget -O /qemu-ppc64le-static https://github.com/multiarch/qemu-user-static/releases/v5.2.0-2/download/qemu-ppc64le-static
```

Comment:
I tested that the wget for the url is not working, the correct url should be:
https://github.com/multiarch/qemu-user-static/releases/tag/v5.2.0-2/download/qemu-ppc64le-static ?

Reply:
yes, done I have updated it.